deepchem/utils/vina_utils.py

```diff
@@ -114,3 +114,72 @@ def load_docked_ligands(
     mol = Chem.MolFromPDBBlock(str(pdb_block), sanitize=False, removeHs=False)
     molecules.append(mol)
   return molecules, scores
+
+
+def prepare_inputs(protein: str, ligand: str,
+                   pdb_name: Optional[str] = None) -> Tuple[RDKitMol, RDKitMol]:
+  """This prepares protein-ligand complexes for docking.
+
+  Autodock Vina requires PDB files for proteins and ligands with
+  sensible inputs. This function uses PDBFixer and RDKit to ensure
+  that inputs are reasonable and ready for docking.
+
```

Comment:
Could you add an examples section here with some usage examples showing how to process a few different protein/ligand systems using this function?

Reply:
How about this:

> Autodock Vina requires PDB files for proteins and ligands with sensible inputs. This function uses PDBFixer and RDKit to attempt to produce inputs that are reasonable and ready for docking. Default values are given for convenience, but fixing PDB files is complicated and human judgement is required to produce protein structures suitable for docking.  Always inspect the results carefully before trying to perform docking.

Reply:
Sounds great, I'll add that in!